README.md

```diff
@@ -145,51 +225,73 @@ module.exports = function(hermione, opts) {
+`SUITE_BEGIN`             | Test suite is about to execute
+`SUITE_END`               | Test suite execution is finished
+`TEST_BEGIN`              | Test is about to execute
+`TEST_END`                | Test execution is finished
+`TEST_PASS`               | Test passed
+`TEST_FAIL`               | Test failed
+`TEST_PENDING`            | Test is skipped
+`RETRY`                   | Test failed but went to retry
+`ERROR`                   | Generic (no tests) errors. For instance, a browser cannot be loaded
+`INFO`                    | Reserved
+`WARNING`                 | Reserved
+`EXIT`                    | Will be triggerd when sigterm is recieved (for example, Ctrl + C). Handler can return a promise.
+
+### mochaOpts
+Extra options for `mocha`
+```javascript
+mochaOpts: {
+    retries: 3
+}
+```
 
-* `specs`(обязательный) - Массив путей до директорий с тестами.
-* `grid` – URL до Selenium grid. По умолчанию `http://localhost:4444/wd/hub`
-* `baseUrl` - Базовый URL тестируемой страницы. По умолчанию `localhost`
-* `timeout` - Время ожидания выполнения теста. По умолчанию `60000`
-* `waitTimeout` - Время ожидания события на странице. По умолчанию `10000`
-* `slow` - Если время выполнения теста превышает это значение, то тест считается медленным. По умолчанию `10000`
-* `debug` - Включает вывод отладочной информации в консоль. По умолчанию `false`
-* `sessionsPerBrowser` - Глобальное значение опции `sessionsPerBrowser` для всех браузеров
-* `retry` - Глобальное значение опции `retry` для всех браузеров
+### prepareBrowser
+Prepare browser session before tests are run. For example, adding custom user commands.
+```js
+prepareBrowser: function(browser) {
+    // do setup here
+}
+```
 
-### Переопределение настроек
+`browser` arguments is a `WebdriverIO` session.
 
-С помощью CLI можно переопределить параметры, используя следующие опции:
+### prepareEnvironment
+Configuration data can be changed depend on extra conditions in `prepareEnvironment` function.
 
-* `-c|--conf` - указать путь к конфигу. По умолчанию используется файл с именем `.hermione.conf.js`
-* `--baseUrl` - задать базовый `url` для всех тестов
-* `--wait-timeout` - время ожидания событий на странице в миллисекундах. По умолчанию 10000
-* `-b|--browser` - запуск тестов в определенном браузере. Возможно задать несколько браузеров одновременно. Например,
+## CLI
 ```
-node_modules/.bin/hermione -b chrome --browser firefox
+  Usage: hermione [options]
+
+  Options:
+
+    -h, --help                 Output usage information
+    -c, --conf <path>          Path to configuration file [./.hermione.conf.js]
+    --baseUrl <url>            Base service url [http://localhost]
+    --grid <url>               Selenium grid URL [http://localhost:4444/wd/hub]
+    --wait-timeout <ms>        Timeout for web page events [10000]
+    --screenshot-path <path>   Path for saving screenshots []
```

Comment:
Why? This is path where webdriver will put screenshots. Another point that now it's not really useful because screenshots will have name like %%date_time%%.png. We already have task to rework this somehow.